src/loader/aarch64/pe/mod.rs

```diff
@@ -168,6 +180,40 @@ impl KernelLoader for PE {
     }
 }
 
+/// load uefi for aarch64
+/// uefi image for virtual machine on arm64 is not formatted, so
+/// a special loader for it is needed.
```

Comment:
I don't think this needs to be part of the public interface documentation.

Reply:
ok, I will remove it.